library/rmc/gen_c_lib.c

```diff
@@ -0,0 +1,55 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
```

Comment:
File level comment: these are stubs to link that make things executable

Reply:
Done.